importer/api/__init__.py

```diff
@@ -4,11 +4,13 @@
 
 from __future__ import unicode_literals
 
+from bs4 import BeautifulSoup
 from shutil import rmtree
 import logging
 from tempfile import mkdtemp
 from os.path import join, exists, isdir
 from os import listdir
+from pprint import pformat
```

Comment:
I do not think there is a proper guideline about the sequence of imports, but we should put the `from ...` all together (maybe after the `import ...`) in the same section (in this case standard libraries)